packages/connector-local/package.json

```diff
@@ -12,6 +12,7 @@
   },
   "description": "hint local connector",
   "devDependencies": {
+    "@hint/parser-html": "^1.0.0",
```

Comment:
Does `connector-local` not depend on this?

Reply:
`connector-local` only depends on receiving the `parse::html::end` event (and will work without it, just obviously without HTML support). The dev dependency is just for the `HTMLParse` type to make TypeScript happy.